.github/workflows/build-release-alpine-nodejs.yml

```diff
@@ -61,28 +81,17 @@ jobs:
     needs: [test]
     runs-on: ubuntu-latest
     steps:
-    - name: Download alpine node.js
+    - name: Download alpine node.js (x64 and arm64)
       uses: actions/download-artifact@v4
       with:
-        name: alpine_nodejs_${{github.event.inputs.NodeVersion}}
-    # Create GitHub release
-    - uses: actions/create-release@master
-      id: createRelease
-      name: Create node.js ${{github.event.inputs.NodeVersion}} Alpine Release
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        pattern: alpine_nodejs_${{github.event.inputs.NodeVersion}}_*
+        merge-multiple: true
+    - name: Release
+      uses: softprops/action-gh-release@v2
       with:
         tag_name: "${{github.event.inputs.NodeVersion}}"
-        release_name: "${{github.event.inputs.NodeVersion}}"
+        name: "${{github.event.inputs.NodeVersion}}"
         body: |
           Alpine node.js ${{github.event.inputs.NodeVersion}}
-    # Upload release assets
-    - name: Upload Release Asset
-      uses: actions/upload-release-asset@v1.0.1
```

Comment:
This action in unmaintained and the repo has been archived: https://github.com/actions/upload-release-asset